databox/client/lib/Function.ts

```diff
@@ -39,9 +48,25 @@ export default class Function<
     return this.components.schema;
   }
 
-  public disableAutorun: boolean;
-  public successCount = 0;
-  public errorCount = 0;
+  public get name(): string {
+    return this.components.name;
+  }
+
+  public get databox(): Databox<any, any> {
+    if (!this.#databox) {
+      this.#databox = new Databox({}, this.databoxInternal);
+      this.databoxInternal.attachFunction(this, null, false);
```

Comment:
Is there no way to attach the actual Databox that has included this Function? Shouldn't something be injecting databoxInternal and databox if they're defined?

Reply:
Yes. #databoxInternal is being inject through attachToDatabox() down below. However, #databox is not, but it should be.

Reply:
Pushed fix to Function and Table.